ComposeSamples/TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/ui/home/HomePage.kt

```diff
@@ -61,9 +68,9 @@ fun SetupUI(windowInfoRep: WindowInfoRepository) {
         viewWidth = LocalConfiguration.current.screenWidthDp / 2
     }
 
-    val isDualScreen = isAppSpanned || isTabletDualMode
+    val isDualScreen = (isAppSpanned || isTabletDualMode) && !isHingeHorizontal
```

Comment:
`isHingeHorizontal` is only useful when app is spanned. Can we have something like what we do for the tablet, such as `val isDualSpannedMode = isAppSpanned && isHingeHorizontal`?